Assets/js/scripts.js

```diff
@@ -130,6 +141,18 @@ form.addEventListener('submit', (event) => {
   }
 });
 
+form.addEventListener('blur', () => {
+  const formData = {
+    first_name: document.getElementById('first_name').value,
```

Comment:
- [ ] Great job so far, instead of using blur as a listener it would be better to update the local storage every time there is a keystroke( key up, keypress, key down) because now if I edit the form without triggering the blur event it'll not update the local storage so kindly change it to the ones I recommended.

Reply:
Now I changed this recommendation